fail2ban/helpers.py

```diff
@@ -137,3 +142,48 @@ def splitcommaspace(s):
 	if not s:
 		return []
 	return filter(bool, re.split('[ ,]', s))
+
+
+class BgService(object):
+	"""Background servicing
+
+	Prevents memory leak on some platforms/python versions, 
+	using forced GC in periodical intervals.
```

Comment:
Might be useful to add URLs here to those issues, so later when we encounter this code we know where need for it comes from.

If "on some" -- should it may be made optional/configuration option?